### ISSUE TITLE
Some trial subscriptions cannot select consumption plan when creating in portal

### ISSUE THREAD

**Issue by anthonychu** [Azure/Azure-Functions#1781]
This has come up a couple of times in recent weeks. Looks like the portal create experience is blocking consumption but it actually works if using other tools like the CLI.

https://github.com/Azure/static-web-apps/issues/207 @GaryParr

/cc @apawast @ehamai

**Comment by weirdyang** [comment]
Hi,
I'm experiencing this issue at the moment, I can't create a function App with plan type of Consumption. Is there a workaround?
![image](https://user-images.githubusercontent.com/25576022/110277938-39f3a400-8011-11eb-92fa-31a717511a46.png)

**Comment by ehamai** [comment]
@shimedh for FYI...

@weirdyang it looks like the region you're testing in the portal (East US 2) is not the same was what you tested in the CLI (West Europe).  Can you confirm if you can create a consumption app in East US 2 please?

**Comment by ehamai** [comment]
Okay I'm dumb.  I completely missed the "trial" qualifier in the title of this issue.  @weirdyang I think you caught us in the middle of the transition to moving trial subs to only consumption plan types which should happen soon.  @shimedh please correct me if I'm remembering this incorrectly.

**Comment by shimedh** [comment]
@ehamai Yes that is correct. We are waiting on Antares deployment to be done before we can make the change in the UX

**Comment by weirdyang** [comment]
Hi,
I can’t create for both. In the CLI I ended up using west europe as I
wasn’t sure of the correct region code for East US 2.
